experimental/tools/elastic-miter/rewrites/a_lhs.mlir

```diff
@@ -1,6 +1,6 @@
 module {
-  handshake.func @a_lhs(%d: !handshake.channel<i32>, %c: !handshake.channel<i1>, ...) -> (!handshake.channel<i32>, !handshake.channel<i32>) attributes {argNames = ["D", "C"], resNames = ["T", "F"]} {
-    %t, %f = cond_br %c, %d {handshake.bb = 1 : ui32, handshake.name = "branch"} : <i1>, <i32>
-    end {handshake.bb = 4 : ui32, handshake.name = "end"} %t, %f : <i32>, <i32>
+  handshake.func @a_lhs(%d: !handshake.channel<i1>, %c: !handshake.channel<i1>, ...) -> (!handshake.channel<i1>, !handshake.channel<i1>) attributes {argNames = ["D_in", "C_in"], resNames = ["T_out", "F_out"]} {
```

Comment:
Regarding argNames/resNames: One character is often a reserved word in NuSMV (pasted the user manual below). The old generator named variables "A_out" if arg name is "A", but the SMV backend names it "A", which causes a syntax error.

![image](https://github.com/user-attachments/assets/3db505e0-728c-44e0-833a-64ce91b96219)